website/docs/docs/cloud/manage-access/set-up-bigquery-oauth.md

```diff
@@ -34,32 +44,22 @@ On the **Credentials** page, you can see your existing keys, client IDs, and ser
 
 Set up an [OAuth consent screen](https://support.google.com/cloud/answer/6158849) if you haven't already. Then, click **+ Create Credentials** at the top of the page and select **OAuth client ID**.
 
-Fill in the application, replacing `YOUR_ACCESS_URL` with the [appropriate Access URL](/docs/cloud/about-cloud/access-regions-ip-addresses) for your region and plan:
-
-| Config | Value |
-| ------ | ----- |
-| **Application type** | Web application |
-| **Name** | dbt Cloud |
-| **Authorized Javascript origins** | https://YOUR_ACCESS_URL |
-| **Authorized Redirect URIs** | https://YOUR_ACCESS_URL/complete/bigquery |
+Fill in the client ID configuration. **Authorized JavaScript Origins** are not applicable. Add an item to **Authorized redirect URIs**, replacing `REDIRECT_URI` with value copied from the connection's **OAuth 2.0 Settings** section in dbt Cloud:
 
+| Config                       | Value           |
+| ---------------------------- | --------------- |
+| **Application type**         | Web application |
+| **Name**                     | dbt Cloud       |
+| **Authorized redirect URIs** | `REDIRECT_URI`  |
 
 Then click **Create** to create the BigQuery OAuth app and see the app client ID and secret values. These values are available even if you close the app screen, so this isn't the only chance you have to save them.
 
-<Lightbox src="/img/docs/dbt-cloud/using-dbt-cloud/dbt-cloud-enterprise/BQ-auth/bq-oauth-app.gif" title="Creating an OAuth app in BigQuery" />
```

Comment:
should this GIF be removed? if this GIF is not being used elsewhere, we can delete it to keep things tidy/clean in our repo

Reply:
yep, I'll delete old images